web-client/src/views/TrialSessionDetails/AllCases.tsx

```diff
@@ -0,0 +1,83 @@
+import { CaseIcons } from '@web-client/ustc-ui/Icon/CaseIcons';
+import { CaseLink } from '../../ustc-ui/CaseLink/CaseLink';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { connect } from '@web-client/presenter/shared.cerebral';
+import { state } from '@web-client/presenter/app.cerebral';
+import React from 'react';
+import classNames from 'classnames';
+
+export const AllCases = connect(
+  {
+    allCases: state.formattedTrialSessionDetails.allCases,
+  },
+  function AllCases({ allCases }) {
+    return (
+      <React.Fragment>
+        <div className="text-right">
+          <span className="text-semibold">Count: </span>
+          {allCases.length}
+        </div>
+        <div className="padding-1"></div>
+        <div className="overflow-x-auto overflow-y-hidden">
```

Comment:
I updated all of these similar files to style and position the Count indicator in the same way, and added scrolling as needed.